Cloud/main.py

```diff
@@ -1,7 +1,110 @@
 #!usr/bin/env python3
 #-*-coding:utf-8-*-
 
+import psycopg2
+import time
+import urllib3
+from edgeRequest import getTemperature
+
+dbname="Cloud"
+username_db="root"
+password_db="root"
+
+default_Location="Paris"
+measure_rate=60
 def run():
-    print("hello world!")
+    urllib3.disable_warnings()
+    connexion = init_db()
+    house_id = getHouseID(connexion)
+
+    print("Starting aquisition on edge, House n"+str(house_id))
+    while(True):
+
+        stats= getTemperature()
+        insert_data(connexion, house_id ,stats)
+        time.sleep(measure_rate)
+    print("Stopping aquisition")
+
+
+        
+    
+
+def init_db():
+    
+   
+    conn = psycopg2.connect(host="localhost",database=dbname, user=username_db, password=password_db)
+    db_cursor = conn.cursor()
+    
+    commands = (
+    """
+        CREATE TABLE IF NOT EXISTS Houses (
+            house_id SERIAL PRIMARY KEY,
+            location VARCHAR(255) NOT NULL
+        )
+        """,
+        """
+        CREATE TABLE IF NOT EXISTS GeoTemp (
+            measure_id SERIAL PRIMARY KEY,
+            house_id SERIAL,
+            temperature_avg FLOAT(5) NOT NULL,
+            temperature_unit CHAR,
+            delta FLOAT(5),
+            clim_duration FLOAT(2),
+            heat_duration FLOAT(2),
+            date VARCHAR(50),
+            FOREIGN KEY (house_id) REFERENCES Houses (house_id)
+        )
+        """)

```

Comment:
Why are you not using script.sql to init database ?

Reply:
Because it allow us to need to manually start the script before starting the program